src/lib/LED/LED.h

```diff
@@ -23,104 +23,143 @@ void WS281BsetLED(uint32_t color)
     strip.ClearTo(RgbColor(HtmlColor(color)));
     strip.Show();
 }
+#endif
+
+#if WS2812_LED_IS_USED || (defined(PLATFORM_ESP32) && defined(GPIO_PIN_LED))
+static enum {
+    STARTUP = 0,
+    NORMAL = 1
+} state;
+static unsigned long blinkyUpdateTime;
```

Comment:
You're introducing a second "updateTime" variable when there already is one in the updateLEDs function used to know the next update time. Just have blinkyUpdate return the Interval it wants the next update to happen at and use the existing static variable in updateLEDs (although interval there will now need to also be a variable and not a const).